crates/symbolicator/src/service.rs

```diff
@@ -383,7 +361,7 @@ impl RequestService {
         f: F,
     ) -> Result<RequestId, MaxRequestsError>
     where
-        F: Future<Output = Result<CompletedResponse, anyhow::Error>> + Send + 'static,
+        F: Future<Output = Result<CompletedResponse>> + Send + 'static,
```

Comment:
```suggestion
        F: Future<Output = anyhow::Result<CompletedResponse>> + Send + 'static,
```

Do we want to use ^ for consistency? We can also do `use anyhowe::Result` in other places instead, I'm fine with both, as long as its consistent.

Reply:
Good point. I believe `anyhow::Result` is a drop-in replacement for `Result`. It is just a type alias with a default Error filled in, so we should be able to just `use anyhow::Result` if we agree on that.

Reply:
Sure, lets do that

Reply:
I replaced some of the cases with a `use anyhow::Result`. For other places where there is only a single use in a file, I rather kept it there.